assets/js/components/UserMenu.js

```diff
@@ -44,52 +49,37 @@ function UserMenu() {
 	const postDisconnectURL = useSelect( ( select ) => select( CORE_SITE ).getAdminURL( 'googlesitekit-splash', { googlesitekit_context: 'revoked' } ) );
 
 	const [ dialogActive, toggleDialog ] = useState( false );
-	const [ menuOpen, toggleMenu ] = useState( false );
-	const menuButtonRef = useRef();
-	const menuRef = useRef();
+	const [ menuOpen, setMenuOpen ] = useState( false );
```

Comment:
renamed `toggleMenu` to `setMenuOpen` because
1. It's like this on `DateRangeSelector`
2. It's not a toggle!